api/v1alpha1/types.go

```diff
@@ -82,6 +84,7 @@ type CombinationStatus struct {
 // +kubebuilder:storageversion
 // +kubebuilder:resource:categories=combo,scope=Cluster
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
```

Comment:
I likely won't have the chance to play around with this locally, but if we're interested in custom printer columns, we may want to demo for the team as well?

Reply:
That would be cool! They're pretty neat, I like them. This is mainly here as guidance for what a custom "status" column would look like (or something of the sort).